client/doc.go

```diff
@@ -0,0 +1,156 @@
+/*
+Package client provides a library for all client-side CONIKS operations.
+
+Introduction
+
+One crucial component of the CONIKS key management system are the clients
+verifying the cryptographic proofs returned by the CONIKS server as part of
+the registration, lookup and monitoring protocols. client implements the
+operations performed by CONIKS clients during these protocols. This document
+outlines each of these protocols from a client perspective.
+
+For the server-side specification of the CONIKS protocols, see
+https://godoc.org/github.com/coniks-sys/coniks-go/server
+
+Registration
+
+Prerequisites: This protocol requires that the client pin both the
+server's public signing key for verifying signed tree roots (STRs) and
+the server's initial private index verification key (i.e. the public
+VRF key). We assume a separate PKI exists to manage the server's keys.
```

Comment:
Why pin the VRF key?  Isn't that in the policy?

> We assume a separate PKI exists to manage the server's keys.

Is this sentence about TLS?